engine/src/test/scala/cromwell/jobstore/JobStoreServiceSpec.scala

```diff
@@ -45,7 +45,7 @@ class JobStoreServiceSpec extends CromwellTestkitSpec with Matchers with Mockito
 
       jobStoreService ! QueryJobCompletion(successKey)
       expectMsgPF(MaxWait) {
-        case ok @ JobComplete(_, JobResultSuccess(Some(0), os)) if os == outputs =>
+        case ok @ JobComplete(JobResultSuccess(Some(0), os)) if os == outputs =>
```

Comment:
This was intentional to try to make it clear this was the expected result, but as it's apparently confusing I'll remove it.